pages/common/irb.md

```diff
@@ -0,0 +1,11 @@
+# irb
+
+> Interactive Ruby Shell.
+
+- Run Ruby Sell with load some gem:
```

Comment:
`with load some gem` doesn't appear to read very well - I'm not sure what you're getting at here?

Reply:
This should be `Launch the shell and load the "net/http" library`